scripts/sync_db.py

```diff
@@ -12,17 +12,44 @@
 dirs = ["database/entries", "database-gba/entries"]
 
 
+def _get_sha1_hash(game, romfile):
+    try:
+        sha1sum = hashlib.sha1()
+
+        with open(f"database/entries/{game}/{romfile}", "rb") as source:
+            block = source.read(2**16)
+
+            while len(block) != 0:
+                sha1sum.update(block)
+                block = source.read(2**16)
+
+        sha1 = sha1sum.hexdigest()
+        print("SHA1:", sha1)
+        return sha1
+
+    except Exception:
+        return ""
+
+
 def run():
     inserted = 0
     updated = 0
+
     for folder in dirs:
         print(f"Processing folder {folder}")
         games = os.listdir(folder)
-        for game in games:
+
+        games_count = len(games)
+        print(f"Found {games_count} games")
+
+        for n, game in enumerate(games, start=1):
             with open(f"{folder}/{game}/game.json") as json_file:
                 data = json.load(json_file)
-                print(f"Processing entry {game}")
-                for file in data["files"]:
+                print(f"({n}/{games_count}) Processing entry {game}")
+
+                romfile = ""
+
+                for file in data.get("files", []):
```

Comment:
Some files do not have a `"files"` key, which led to `KeyErrors` (e.g. `wrong-road-gba`)

Reply:
All the entries from the gb database have a files key, it's mandatory per the schema, but I still didn't add CI/CD to the gba repository, so it's that entry that has the problem, not the script.